libs/vkd3d/vkd3d_private.h

```diff
@@ -1876,6 +1876,22 @@ struct vkd3d_query_range
     uint32_t flags;
 };
 
+enum vkd3d_rendering_flags
+{
+    VKD3D_RENDERING_ACTIVE    = (1u << 0),
```

Comment:
I'm struggling a bit reasoning about these flags. From what I understand from earlier:
- ACTIVE: We're inside CmdBeginRenderingKHR.
- SUSPENDED: We had to do non-rendering, so we called EndRenderingKHR. However, we did not perform any kind of "end pass barriers".
- CURRENT: Not sure what this represents anymore.

Reply:
`CURRENT` is a dirty flag for the rendering info.